ui/src/pages/login/Login.tsx

```diff
@@ -250,24 +219,38 @@ const PartiesLoginForm: React.FC<Props> = ({ onLogin }) => {
     <>
       <p className="login-details dark">
         <span>
-          Alternatively, login with <code className="link">parties.json</code> located in the DABL
-          Console Users tab:{' '}
+          Alternatively, login with <code className="link">parties.json</code> located in the Daml
+          Hub Console Identities tab:
         </span>
       </p>
       <FormErrorHandled size="large" className="login-form" onSubmit={handleLogin}>
         {loadAndCatch => (
           <>
             <Form.Group widths="equal">
               <Form.Input className="upload-file-input">
-                <label className="custom-file-upload button ui">
-                  <DablPartiesInput
-                    ledgerId={ledgerId}
-                    onError={error => loadAndCatch(handleError(error))}
-                    onLoad={handleLoad}
-                  />
-                  <Icon name="file" className="white" />
-                  <p className="dark">Load Parties</p>
-                </label>
+                <DamlHubLogin
+                  options={{
+                    method: {
+                      file: {
+                        render: () => (
+                          <label className="custom-file-upload button ui">
+                            {' '}
```

Comment:
these sneak in sometimes...

Reply:
I've noticed this too. It's a thing that Prettier does to preserve spaces, if you have some JSX/HTML that spans multiple lines, then whitespace is collapsed. So something like 

```
<p>Hello 
    <a>world</a>
</p>
```

would render on the browser as `Helloworld`. Thus Prettier would change that to 

```
<p>
    Hello {' '} 
    <a>world</a>
</p>
```

so the explicit space is preserved. In _this_ case, there's no need for it because there's no text at the start, but I probably had a whitespace there in the code before the formatting, and Prettier overzealously preserved it (you see it happened to this guy too https://stackoverflow.com/questions/64358015/why-does-prettier-automatically-insert-empty-interpolation-code-characters)